app/src/main/java/fr/free/nrw/commons/notification/NotificationRenderer.java

```diff
@@ -60,7 +66,25 @@ private void setTitle(String notificationText) {
         notificationText = notificationText.trim().replaceAll("(^\\h*)|(\\h*$)", "");
         notificationText = Html.fromHtml(notificationText).toString();
         notificationText = notificationText.concat(" ");
-        title.setText(notificationText);
+
+        SpannableString ss = new SpannableString(notificationText);
+        ClickableSpan clickableSpan = new ClickableSpan() {
+            @Override
+            public void onClick(View view) {
+                listener.notificationClicked(getContent());
+            }
+
+            @Override
+            public void updateDrawState(TextPaint ds) {
+                super.updateDrawState(ds);
+                ds.setUnderlineText(false);
+                ds.setColor(Color.BLACK);
+            }
+        };
+
+        ss.setSpan(clickableSpan, 0, notificationText.length(), Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
```

Comment:
Could you add a comment explaining the magic number "0" or make it a constant with an informative name?

Reply:
Ok, I will add a comment.